### DISCUSSION TITLE
About saving the session state

### DISCUSSION THREAD

**Discussion by viper0078** [cefsharp/CefSharp#4412] [Q&A]
C# .net FrameWork4.7.2 CefSharp 110.0.300.0 
We will place one Chromiumwebbrowser in Form.
If I log in to Yahoo with the Chromiumwebbrowser, I want to maintain the session the next time I run it.
I assume it uses cookies, but how should I implement this?
I'm looking for sample sources, but I couldn't run them because it seems some functions are no longer supported in CefSharp 110.0.300.0.
Thank you in advance for your help.

**Comment by amaitland** [comment]
> Specifying a CachePath is required for persistence of cookies, saving of passwords, etc, an In-Memory cache is used by default (similar to Incogneto). See [Initialize and Shutdown](https://github.com/cefsharp/CefSharp/wiki/General-Usage#initialize-and-shutdown) section below for an example of Initializing CEF with a CachePath below.

As per https://github.com/cefsharp/CefSharp/wiki/General-Usage#need-to-knowlimitations you need to set [CefSettings.CachePath](https://cefsharp.github.io/api/110.0.x/html/P_CefSharp_CefSettingsBase_CachePath.htm)

> I'm looking for sample sources, but I couldn't run them because it seems some functions are no longer supported

See https://github.com/cefsharp/CefSharp.MinimalExample for a working example (this is always kept up to date).
